sycl/doc/syclcompat/README.md

```diff
@@ -742,6 +742,10 @@ sycl::queue create_queue(bool print_on_async_exceptions = false,
 // device manager.
 sycl::queue get_default_queue();
 
+// Util function to set the default queue of the current device in the
+// device manager.
+void set_default_queue(const sycl::queue &q);
```

Comment:
It would be good to document that calling this function will overwrite the `saved_queue`, and that this function may be blocking (if you add the code to flush the old default queue)